x-pack/test_serverless/api_integration/test_suites/common/saved_objects_management/bulk_get.ts

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { SavedObjectWithMetadata } from '@kbn/saved-objects-management-plugin/common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { RoleCredentials } from '../../../../shared/services';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const svlUserManager = getService('svlUserManager');
+  const supertest = getService('supertest');
```

Comment:
Same as above, should be `const supertestWithoutAuth = getService('supertestWithoutAuth');`

Reply:
[`78cc092`](https://github.com/elastic/kibana/pull/184888/commits/78cc09208663b89dc67746e33ac8c78d5f6e741f)